sklearn/cluster/tests/test_mean_shift.py

```diff
@@ -55,6 +55,36 @@ def test_mean_shift():
     assert_equal(n_clusters_, n_clusters)
 
 
+def test_mean_shift_negative_bandwidth():
+    bandwidth = -1
+    ms = MeanShift(bandwidth=bandwidth)
+    msg = \
+        "bandwidth needs to be greater than zero or None," \
+        "            got -1.000000"
+    assert_raise_message(ValueError, msg, ms.fit, X)
+
+
+def test_seeds():
```

Comment:
I don't get what this is testing. Checking that parameters are maintained should usually be covered by common tests not tests for each specific estimator

Reply:
removed